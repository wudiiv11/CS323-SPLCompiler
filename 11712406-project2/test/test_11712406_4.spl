struct string {
  int size;
  int value[256];
};

int strcpy(struct string s1, struct string s2) {
    int i = 0;
    while (i < 256) {
        s1.value[i] = s2.value[i];
        i = i + 1;
    }
    s1.size = s2.size;
    return string.size + i * 5;
}

int main() {
    struct string from;
    strcpy(from, 1);
}