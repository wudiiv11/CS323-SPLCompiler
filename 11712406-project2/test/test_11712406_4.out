Error type 9 at Line 18: function's args mismatch the declared para